jacobian_follower/include/jacobian_follower/jacobian_follower.hpp

```diff
@@ -100,6 +102,10 @@ class JacobianFollower
                   double max_velocity_scaling_factor,
                   double max_acceleration_scaling_factor);
 
+  std::vector<std::vector<double>> compute_IK_solutions(geometry_msgs::Pose target_pose, const std::string& group_name);
```

Comment:
It's unclear how the order of joints gets determined here. Maybe put it in a comment, or return vector of JointState messages?

Reply:
I'm all for clarity, especially when going between python and C++.

What situation are you thinking of where the order would be ambiguous? My assumption is "group_name" has a single FK chain and that (by convention) joints are in order of base -> EE.

Reply:
I never assume that's the order. Sometimes things are ordered by chain, sometimes alphabetically, sometimes because someone write them in some particular order somewhere. But even if it were obvious, as you elude might not be a chain. It should be easy to return a  vector of JointState messages?

Reply:
I added a TODO